packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -24,7 +24,7 @@ import 'package:flutter/foundation.dart';
 /// done before calling [runApp].
 ///
 /// To determine whether a draw operation is useful for warming up shaders,
-/// check whether it improves the slowest GPU frame. Also, tracing with
+/// check whether it improves the slowest raster frame. Also, tracing with
```

Comment:
Nit: `slowest frame rasterization time` might be better than `slowest raster frame`.

Reply:
Ack, will fix.